app/liquid/liquid_renderer.rb

```diff
@@ -26,7 +26,7 @@ def default_markup
   end
 
   def data
-    @data ||= Plugins.data_for_view(@page, {form_values: @member.try(:attributes)}).
+    @data ||= Plugins.data_for_view(@page, {form_values: @member.try(:attributes), url_params: @url_params}).
```

Comment:
Yeah, I think you're right, that would be the better way to do it. Going to change.